examples/prisma/app/root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration
+} from "remix";
+
```

Comment:
Could you add a loader/action here that allows people to do some basic CRUD (minimal is better). I think it would be helpful for people to see what it takes to make things work together with Remix.